pkg/api/v1alpha1/nutanixdatacenterconfig.go

```diff
@@ -19,6 +21,10 @@ func NewNutanixDatacenterConfigGenerate(clusterName string) *NutanixDatacenterCo
 		Spec: NutanixDatacenterConfigSpec{
 			Endpoint: "<enter Prism Central Endpoint (FQDN or IP) here>",
 			Port:     9440,
+			CredentialRef: &Ref{
+				Kind: constants.SecretKind,
+				Name: constants.NutanixCredentialsName,
```

Comment:
Not sure if it was mentioned in the design doc, why are we using a constant for the credRef instead of for example, clusterName?

Reply:
Cluster name is the secret corresponding to the cluster and consumed by CAPX (one secret per cluster) that EKS-A will generate. The EKS-A secrets are one per datacenter config. The name could be whatever but we need a dummy/default value hence "nutanix-credentials" making it a bit more obvious what it is.